department-of-veterans-affairs/caseflow
# FactoryBot: randomize Veterans' last name

Bumps #15812

### Description
Use different names besides "Bob Smith" to uniquely identify Veterans with different file numbers. 
Fix some tests that rely on "Bob Smith".

## Impact
For users and developers, this PR improves confidence that Caseflow is working correctly with unique names and tests are not reliant on FactoryBot-generated Veterans having the same name.

### Acceptance Criteria
- [ ] Code compiles correctly; tests pass

## Testing
1. Run `make reset` and waiting a long while until it's complete.
1. View various cases in the UI. Should see veterans with names starting with "Bob Smith" but different last names.
![image](https://user-images.githubusercontent.com/55255674/105559888-fef31680-5cd7-11eb-9ce3-220b7e5a25dc.png)
